packages/editor/src/utils/media-upload/media-upload.js

```diff
@@ -68,7 +68,7 @@ export function mediaUpload( {
 
 	// Allowed type specified by consumer
 	const isAllowedType = ( fileType ) => {
-		return ( allowedType === '*' ) || startsWith( fileType, `${ allowedType }/` );
```

Comment:
Aren't we breaking support for the `'*'` wildcard by removing it here? Was that in use anywhere, or documented? Also saddening if this is not breaking any tests with its removal, since it means it is not well-covered by tests.

Reply:
If `*` was the wild card desired couldn't that be defined in the allow type argument of mediaPlaceholder as `allowedType` which would get passed here? Reverting this line causes successful type check uploads to fail for Playlists...ill have to dig into that if we want to revert this. the Files block uses `*` and I have been testing that on in tandem with making the playlist block. Thoughts?

Reply:
I'm not really sure I understand.

- What failures were you seeing without these changes? You're not specifying `*` so I don't see why it would have any impact to leave it.
- Regardless of whether a `*` is sensible, it exists and is already in use by the Files block. If we want to remove it, we need to remove any references to it. Speaking personally, I don't really like `*` as the wildcard. It seems like if someone wants any file to be uploaded, they should just omit the `allowedType` property altogether. But then again, this change has nothing to do with a Playlist block implementation and seems like it should be a separate effort.

Reply:
I've reverted this line and uploads no longer work for playlist only...going to have to dig in to figure out whats going wrong. I cant even get a logging of `fileType` now. For what it's worth, the file block was accepting all types previously before reverting the change